targets/TARGET_STM/TARGET_STM32F0/serial_device.c

```diff
@@ -468,49 +468,21 @@ void serial_putc(serial_t *obj, int c)
     huart->Instance->TDR = (uint32_t)(c & (uint16_t)0xFF);
 }
 
-int serial_readable(serial_t *obj)
-{
-    struct serial_s *obj_s = SERIAL_S(obj);
-    UART_HandleTypeDef *huart = &uart_handlers[obj_s->index];
-    
-    // Check if data is received
-    return (__HAL_UART_GET_FLAG(huart, UART_FLAG_RXNE) != RESET) ? 1 : 0;
-}
-
-int serial_writable(serial_t *obj)
-{
-    struct serial_s *obj_s = SERIAL_S(obj);
-    UART_HandleTypeDef *huart = &uart_handlers[obj_s->index];
-    
-    // Check if data is transmitted
-    return (__HAL_UART_GET_FLAG(huart, UART_FLAG_TXE) != RESET) ? 1 : 0;
-}
-
 void serial_clear(serial_t *obj)
 {
     struct serial_s *obj_s = SERIAL_S(obj);
     UART_HandleTypeDef *huart = &uart_handlers[obj_s->index];
-    
+
     huart->TxXferCount = 0;
     huart->RxXferCount = 0;
 }
 
-void serial_pinout_tx(PinName tx)
-{
-    pinmap_pinout(tx, PinMap_UART_TX);
-}
-
 void serial_break_set(serial_t *obj)
 {
     struct serial_s *obj_s = SERIAL_S(obj);
     UART_HandleTypeDef *huart = &uart_handlers[obj_s->index];
-    
-    //HAL_LIN_SendBreak(huart);
-}
 
-void serial_break_clear(serial_t *obj)
-{
-    (void)obj;
+    //HAL_LIN_SendBreak(huart);
```

Comment:
that's a good question indeed ;-) 
The diff somehow displays even though I haven't changed this line at all - it was there already before and I decided to leave it for now as this is not directly related to this PR